### PR TITLE
docs: Update Cube Cloud home docs

### PR DESCRIPTION
**Pull request by keydunov**

### CODE REVIEW

File: docs/content/Cube-Cloud/Index.md [hunk 1]
```diff
@@ -5,10 +5,41 @@ category: Overview
...
 performance. It includes all the core features of Cube.js, while taking care of
 infrastructure concerns such as the number of instances, memory,
 high-availability, pre-aggregations management, caching, scalability, real-time
 monitoring and tracing.
 
 [Quickstart](/cloud/quickstart)
+
+## Develop Cube.js projects
+
+You can use Cube Cloud IDE to develop and run Cube.js applications. By connecting your Github account, you can keep your data schema under version control.
```
Comment (rpaik) [inline docs/content/Cube-Cloud/Index.md:26]:
```suggestion
You can use Cube Cloud IDE to develop and run Cube.js applications. By connecting your GitHub account, you can keep your data schema under version control.
```

File: docs/content/Cube-Cloud/Index.md [hunk 2]
```diff
@@ -5,10 +5,41 @@ category: Overview
...
+
+![](https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Cube-Cloud/cube-ide.png)
+
+## Run and Scale Cube.js applications
+
+Cube Cloud provides on-demand scalable infrastructure and pre-aggregations storage. Cube Cloud run hundreds of Cube Store instances to ingest and query pre-aggregations with low latency and high concurrency. It is available to all users on the Standard plan and higher.
```
Comment (rpaik) [inline docs/content/Cube-Cloud/Index.md:32]:
```suggestion
Cube Cloud provides on-demand scalable infrastructure and pre-aggregations storage. Cube Cloud runs hundreds of Cube Store instances to ingest and query pre-aggregations with low latency and high concurrency. It is available to all users on the Standard plan and higher.
```

**Review by rpaik — APPROVED** [review]
Found a couple of minor typos 😄

**Comment by keydunov** [comment]
Thank you @rpaik!

**Pull request merged**